index.d.ts

```diff
@@ -4,4 +4,4 @@ declare function run(
   probotApp: ApplicationFunction | ApplicationFunction[]
 ): Promise<void>;
 
-export = { run };
+export default run;
```

Comment:
we don't export `run` as default export though. The usage is

```js
const { run } = require('@probot/adapter-github-actions')
```

Reply:
I see, I now remember why I did it like this.

I just wanted to use `import from` syntax, but `require` works for me.

Reply:
you can use `import { run } from "@probot/adapter-github-actions"

Reply:
When I use this approach, I'm getting:
```
module "node_modules/@probot/adapter-github-actions/index"
This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.ts(2497)
```

Reply:
I'm not sure what it means, I'd have to look it up myself and I don't have the time right now, sorry

Reply:
That's ok, Thanks Gregor!